jdaviz/main_styles.vue

```diff
@@ -213,6 +213,10 @@ a:active {
   background-color: #c7510996 !important;
 }
 
+.vuetify-styles .v-btn-toggle>.v-btn.v-btn {
+    opacity: 1;
+}
```

Comment:
I'm not sure we want this for _all_ buttons, do we?  Can we put it in the existing `.jdaviz-nested-toolbar .v-btn, .plugin-nested-toolbar .v-btn` (or perhaps we don't even want it for the plugin toolbars, I'm not sure, that might be a question for @Jenneh)?

Reply:
Let me know if the latest commit is more what you had in mind.

Reply:
that works, but is there a reason the _existing_ entry doesn't do the necessary filter?  It just might be easier for future maintenance to keep it all bundled together

Reply:
Do you mean like this?
```
.jdaviz-nested-toolbar .v-btn, .plugin-nested-toolbar .v-btn, .jdaviz-nested-toolbar .v-btn-toggle>.v-btn.v-btn {
  height: 42px !important;
  border: none !important;
  min-width: 42px !important;
  /* remove "dimming" since we use orange background for active */
  color: transparent ! important;
  opacity: 1;
}
```
I need to add the `.jdaviz-nested-toolbar .v-btn-toggle>.v-btn.v-btn` for it to work.

Reply:
hmm it works for me with the existing selector (without having to add `.jdaviz-nested-toolbar .v-btn-toggle>.v-btn.v-btn`) 🤔 

But if that doesn't work on your end, then I guess a separate entry is fine.